moveit_simple/test/moveit_simple.cpp

```diff
@@ -62,7 +62,21 @@ TEST(MoveitSimpleTest, add_trajectory)
   EXPECT_TRUE(robot.addTrajPoint(TRAJECTORY_NAME, "waypoint2", 3.0));
   EXPECT_TRUE(robot.addTrajPoint(TRAJECTORY_NAME, "waypoint3", 4.0));
 
-  EXPECT_TRUE(robot.execute(TRAJECTORY_NAME));
+  // Check if a proper plan and execution works.
+  moveit_simple::TrajectoryGoal trajectory_goal;
+  EXPECT_TRUE(robot.plan(TRAJECTORY_NAME, trajectory_goal));
```

Comment:
@shaun-edwards ! Won't it be easier for user to execute the trajectory by just passing the traj_name? trajectory_goal can be passed internally by using a simple map

 < std::map<std::string, TrajectoryInfo> traj_info_map_>